var/spack/repos/builtin/packages/libtool/package.py

```diff
@@ -3,7 +3,7 @@
 class Libtool(Package):
     """libtool -- library building part of autotools"""
     homepage = "https://www.gnu.org/software/libtool/"
-    url      = "http://ftpmirror.gnu.org/libtool/libtool-2.4.2.tar.gz"
+    url      = "http://ftpmirror.gnu.org/libtool/libtool-2.4.6.tar.gz"
```

Comment:
the original version should work, AFAIK. So should happen is that the pattern is picked up from the `url` and then substituted using the currently chosen version. Does it not work for you without this change and `2.4.6`?

p.s. sometimes url parser fails, like for `tbb` and one has to give an url for each version https://github.com/LLNL/spack/blob/develop/var/spack/repos/builtin/packages/tbb/package.py#L14 . But here it looks easy and is supposed to work.